tst/standard/attr.tst

```diff
@@ -316,6 +316,23 @@ gap> DigraphStronglyConnectedComponents(gr2);
 rec( comps := [ [ 1 ], [ 2 ], [ 3 ], [ 4 ], [ 5 ], [ 6 ], [ 7 ], [ 8 ], 
       [ 9 ], [ 10 ] ], id := [ 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 ] )
 
+#  DigraphNrStronglyConnectedComponents
+gap> D := CycleDigraph(10);;
+gap> for i in [1 .. 1000] do
+> D := DigraphDisjointUnion(D, CycleDigraph(10));
+> od;
+gap> DigraphNrStronglyConnectedComponents(D);
+1001
+gap> D := CayleyDigraph(SymmetricGroup(6));;
+gap> DigraphNrStronglyConnectedComponents(D);
+1
+gap> D := Digraph([[2, 3], [1, 4, 5], [3], [2, 5], [6], [3, 5]]);;
+gap> DigraphNrStronglyConnectedComponents(D);
+3
+gap>  D := Digraph([[]]);;
+gap> DigraphNrStronglyConnectedComponents(D);
+1
```

Comment:
Please also add a test for `DigraphNrStronglyConnectedComponents(EmptyDigraph(0));` 🙂